rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -39,10 +39,12 @@
   public static final int DEFAULT_BOOKKEEPER_SERVER_PORT = 8899;
   public static final int DEFAULT_DATA_CACHE_FULLNESS = 80; // percent
 
+  private static final String KEY_USE_CUSTOM_REPORTER = "rubix.custom.metrics.enabled";
```

Comment:
Why is the boolean flag needed ?
Isn't the presence of a non-default `rubix.metric-collector.impl` enough ?

Reply:
removed